tools/publish_yaml_data/verify_assets_reachable.py

```
@@ -51,61 +59,80 @@
-    pairs: list[tuple[str, str]] = []
+    assets: list[tuple[str, str, str]] = []
     for entry in files:
         if not isinstance(entry, dict):
             raise SystemExit(f"FAIL: manifest entry is not a mapping: {entry!r}")
         name = entry.get("name")
         url = entry.get("download_url")
+        digest = entry.get("sha256")
         if not isinstance(name, str) or not isinstance(url, str):
             raise SystemExit(
                 f"FAIL: manifest entry missing string name/download_url: {entry!r}"
             )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)